cunumeric/config.py

```diff
@@ -769,14 +772,32 @@ def reverse(in_string: Union[str, None]) -> str:
 
 
 # Match these to CuNumericTypeCodes in cunumeric_c.h
-@unique
-class CuNumericTypeCodes(IntEnum):
-    CUNUMERIC_TYPE_POINT1 = _cunumeric.CUNUMERIC_TYPE_POINT1
-    CUNUMERIC_TYPE_POINT2 = _cunumeric.CUNUMERIC_TYPE_POINT2
-    CUNUMERIC_TYPE_POINT3 = _cunumeric.CUNUMERIC_TYPE_POINT3
-    CUNUMERIC_TYPE_POINT4 = _cunumeric.CUNUMERIC_TYPE_POINT4
-    CUNUMERIC_TYPE_POINT5 = _cunumeric.CUNUMERIC_TYPE_POINT5
-    CUNUMERIC_TYPE_POINT6 = _cunumeric.CUNUMERIC_TYPE_POINT6
-    CUNUMERIC_TYPE_POINT7 = _cunumeric.CUNUMERIC_TYPE_POINT7
-    CUNUMERIC_TYPE_POINT8 = _cunumeric.CUNUMERIC_TYPE_POINT8
-    CUNUMERIC_TYPE_POINT9 = _cunumeric.CUNUMERIC_TYPE_POINT9
+# we start from POINT2 type since POINT1 is int8 type
+_CUNUMERIC_DTYPES = [
```

Comment:
Once https://github.com/nv-legate/legate.core/pull/333 is merged, I don't think you need to use the internal class `_Dtype` anymore. I think it will be enough to store these as 3-tuples.

Reply:
Thank you, I will change the implementation accordingly